modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -90,6 +96,10 @@ class CondaCache {
 
         if( config.cacheDir )
             configCacheDir0 = (config.cacheDir as Path).toAbsolutePath()
+
+        if( config.useMamba )
+            useMamba = config.useMamba as Boolean
```

Comment:
`boolean` instead of `Boolean` to match the type declaration

Reply:
Copy that 👍